<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateAddrDialogClass</class>
 <widget class="QDialog" name="UpdateAddrDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Обновление адреса</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Текущий адрес устройства</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="cur_addr_spin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>254</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Сменить адрес на</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="targ_addr_spin">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>254</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="upd_btn">
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="cancel_btn">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
